src/App.js

```diff
@@ -3,13 +3,22 @@ import { connect } from 'react-redux';
 import './App.css';
 import { GameForm } from './components/GameForm';
 import { Board } from './components/Board';
-import { GAME_SETUP, PLAY } from './rootReducer';
+import { GameSummary } from './components/GameSummary';
+import { Timer } from './components/Timer';
+import { GAME_SETUP, PLAY, GAME_END } from './rootReducer';
 
 const AppPure = ({ phase }) => {
```

Comment:
Będą jeszcze jakieś inne fazy gry?

Reply:
nie, tylko tyle

Reply:
To ok, zostaw bo jeżeli doszedł by jeszcze jeden przypadek to było by to już zbyt skomplikowane ;)